bin/faucet

```diff
@@ -4,5 +4,7 @@
 let { faucetDispatch } = require("../lib");
 let { parseCLI } = require("../lib/cli");
 
-let { referenceDir, config, options } = parseCLI();
-faucetDispatch(referenceDir, config, options);
+parseCLI().
```

Comment:
Top Level await is only available in ESMs, right?

Reply:
yup: didn't seem worth creating a hoisted function for in this case

(for posterity: top-level `await` was [introduced](https://nodejs.org/en/blog/release/v14.8.0/) in Node v14.8)